app/lucy/src/app/components/nav-bar/nav-bar.component.ts

```diff
@@ -265,34 +265,34 @@ export class NavBarComponent implements OnInit, AfterViewInit, OnDestroy {
    * Navigate to Profile Component
    */
   navigateToProfile() {
-    this.routerService.navigateTo(AppRoutes.Profile);
+    this.routerService.navigateTo(AppRoutes.Profile, undefined, true);
   }
 
   /**
    * Navigate to Admin Tools Component
    */
   navigateToAdminTools() {
-    this.routerService.navigateTo(AppRoutes.AdminTools);
+    this.routerService.navigateTo(AppRoutes.AdminTools, undefined, true);
   }
 
   /**
    * Navigate to Add Observation Component
    */
   navigateToAddEntry() {
-    this.routerService.navigateTo(AppRoutes.AddEntry);
+    this.routerService.navigateTo(AppRoutes.AddEntry, undefined, true);
```

Comment:
Undefined as method parameter is not recommended. So you can update method code and accept null as parameter instead of undefined